common/src/main/templates/io/netty/util/collection/KObjectHashMap.template

```diff
@@ -627,7 +625,7 @@ public class @K@ObjectHashMap<V> implements @K@ObjectMap<V> {
 
         @Override
         public void remove() {
-            if (prevIndex < 0) {
+            if (prevIndex == -1) {
```

Comment:
I think so. its either set to `-1` at init time, an index in an array (non-negative), or `-1` if the previous index has been removed.

Am I missing something?